libraries/ESP8266httpUpdate/src/ESP8266httpUpdate.cpp

```diff
@@ -276,7 +304,7 @@ HTTPUpdateResult ESP8266HTTPUpdate::handleUpdate(HTTPClient& http, const String&
                 ret = HTTP_UPDATE_FAILED;
             } else {
 
-                WiFiClient * tcp = http.getStreamPtr();
+                WiFiClient * tcp = (WiFiClient *) http.getStreamPtr();
```

Comment:
I'm fine with casts like this, but I think @devyte will probably ask for static_cast<>.  No change from me, just a warning...

Reply:
I will try the static_cast

Reply:
Because http.getStreamPtr() returns a WiFiClient now, casting is no longer necessary